lms/templates/courseware/courseware.html

```diff
@@ -25,13 +25,27 @@
     <%static:include path="common/templates/${template_name}.underscore" />
 </script>
 % endfor
+
 % if settings.FEATURES.get('ENABLE_COURSEWARE_SEARCH'):
     % for template_name in ["course_search_item", "course_search_results", "search_loading", "search_error"]:
-    <script type="text/template" id="${template_name}-tpl">
-        <%static:include path="search/${template_name}.underscore" />
-    </script>
-% endfor
+        <script type="text/template" id="${template_name}-tpl">
+            <%static:include path="search/${template_name}.underscore" />
+        </script>
+    % endfor
 % endif
+
+% for template_name in ["proctored-exam-status"]:
```

Comment:
Should this only be conditionally added-- if settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False) and course.enable_proctored_exams?